src/device/device_ledger.cpp

```diff
@@ -303,7 +303,29 @@ namespace hw {
       offset += strlen(MONERO_VERSION);
       this->buffer_send[4] = offset-5;
       this->length_send = offset;
-      this->exchange();      
+      this->exchange();
+
+      ASSERT_X(this->length_recv>=3, "Communication error, less than tow three bytes received. Check your application version.");
```

Comment:
tow ?

Reply:
major, minor, micro : three

Reply:
Yes, I mean the "tow" seems like it's extra unneeded word here.

Reply:
OMG yes. It seems I'am a bit tired :sleepy: